### PR TITLE
Added more Polish translations

### PR DESCRIPTION
**Pull request by Maciox55**
Added a lot of translation for Polish, only a handful is missing.

### CODE REVIEW

File: addons/camera/stringtable.xml [hunk 1]
```diff
@@ -24,6 +27,7 @@
         <Key ID="STR_ZEN_Camera_DefaultSpeedCoef">
             <English>Default Speed Coefficient</English>
             <German>Standardmäßiger Geschwindigkeitskoeffizient</German>
+            <Polish>Współczynnik prędkości domyślnej </Polish>
```
Comment (neilzar) [inline addons/camera/stringtable.xml]: Trailing space
```suggestion
            <Polish>Współczynnik prędkości domyślnej</Polish>
```

File: addons/attributes/stringtable.xml [hunk 2]
```diff
@@ -38,38 +38,47 @@
...
         </Key>
         <Key ID="STR_ZEN_Attributes_ChangeSide">
             <English>Change Side</English>
             <German>Seite ändern</German>
+            <Polish>Zmień Stronę</Polish>
         </Key>
         <Key ID="STR_ZEN_Attributes_ChangeSkills">
             <English>Change Skills</English>
             <German>Fähigkeiten ändern</German>
+            <Polish>Zmień </Polish>
```
Comment (neilzar) [inline addons/attributes/stringtable.xml]: Trailing space
```suggestion
            <Polish>Zmień</Polish>
```

File: addons/attributes/stringtable.xml [hunk 3]
```diff
@@ -38,38 +38,47 @@
...
         </Key>
         <Key ID="STR_ZEN_Attributes_ChangeSkills">
             <English>Change Skills</English>
             <German>Fähigkeiten ändern</German>
+            <Polish>Zmień</Polish>
         </Key>
         <Key ID="STR_ZEN_Attributes_Skills">
             <English>Skills</English>
             <German>Fähigkeiten</German>
+            <Polish>Umiejętnośc</Polish>
```
Comment (veteran29) [inline addons/attributes/stringtable.xml]:
```suggestion
            <Polish>Umiejętności</Polish>
```

File: addons/attributes/stringtable.xml [hunk 4]
```diff
@@ -38,38 +38,47 @@
...
         </Key>
         <Key ID="STR_ZEN_Attributes_ChangeSide">
             <English>Change Side</English>
             <German>Seite ändern</German>
+            <Polish>Zmień Stronę</Polish>
         </Key>
         <Key ID="STR_ZEN_Attributes_ChangeSkills">
             <English>Change Skills</English>
             <German>Fähigkeiten ändern</German>
+            <Polish>Zmień</Polish>
```
Comment (veteran29) [inline addons/attributes/stringtable.xml]:
```suggestion
            <Polish>Zmień umiejętności</Polish>
```

File: addons/area_markers/stringtable.xml [hunk 5]
```diff
@@ -4,14 +4,17 @@
         <Key ID="STR_ZEN_Area_Markers_CreateAreaMarker">
             <English>Create Area Marker</English>
             <German>Bereichsmarkierungen erstellen</German>
+            <Polish>Stwórz Marker</Polish>
         </Key>
         <Key ID="STR_ZEN_Area_Markers_EditAreaMarker">
             <English>EDIT AREA MARKER</English>
             <German>BEREICHSMARKIERUNG BEARBEITEN</German>
+            <Polish>Edytuj Marker</Polish>
```
Comment (veteran29) [inline addons/area_markers/stringtable.xml]:
```suggestion
            <Polish>EDYTUJ MARKER OBSZAROWY</Polish>
```

File: addons/ai/stringtable.xml [hunk 6]
```diff
@@ -3,10 +3,12 @@
     <Package name="AI">
         <Key ID="STR_ZEN_AI_Fastrope">
             <English>Fastrope</English>
+            <Polish>Szybki desant z liny</Polish>
         </Key>
         <Key ID="STR_ZEN_AI_Paradrop">
             <English>Paradrop</English>
             <German>Fallschirmsprung</German>
+            <Polish>Desasnt ze spadochronem</Polish>
```
Comment (veteran29) [inline addons/ai/stringtable.xml]:
```suggestion
            <Polish>Desant ze spadochronem</Polish>
```

File: addons/ai/stringtable.xml [hunk 7]
```diff
@@ -3,10 +3,12 @@
     <Package name="AI">
         <Key ID="STR_ZEN_AI_Fastrope">
             <English>Fastrope</English>
+            <Polish>Szybki desant z liny</Polish>
```
Comment (veteran29) [inline addons/ai/stringtable.xml]:
```suggestion
            <Polish>Zjazd na linach</Polish>
```

File: addons/area_markers/stringtable.xml [hunk 8]
```diff
@@ -4,14 +4,17 @@
         <Key ID="STR_ZEN_Area_Markers_CreateAreaMarker">
             <English>Create Area Marker</English>
             <German>Bereichsmarkierungen erstellen</German>
+            <Polish>Stwórz Marker</Polish>
```
Comment (veteran29) [inline addons/area_markers/stringtable.xml]:
```suggestion
            <Polish>Stwórz Marker Obszarowy</Polish>
```

File: addons/attributes/stringtable.xml [hunk 9]
```diff
@@ -38,38 +38,47 @@
...
         </Key>
         <Key ID="STR_ZEN_Attributes_Skills">
             <English>Skills</English>
             <German>Fähigkeiten</German>
+            <Polish>Umiejętnośc</Polish>
         </Key>
         <Key ID="STR_ZEN_Attributes_AimingAccuracy">
             <English>Aiming Accuracy</English>
             <German>Zielgenauigkeit</German>
+            <Polish>Precyzja Wycelowania</Polish>
```
Comment (veteran29) [inline addons/attributes/stringtable.xml]:
```suggestion
            <Polish>Celność</Polish>
```

File: addons/attributes/stringtable.xml [hunk 10]
```diff
@@ -38,38 +38,47 @@
...
         </Key>
         <Key ID="STR_ZEN_Attributes_AimingSpeed">
             <English>Aiming Speed</English>
             <German>Zilelgeschwindigkeit</German>
+            <Polish>Prędkość Wycelowania</Polish>
```
Comment (veteran29) [inline addons/attributes/stringtable.xml]:
```suggestion
            <Polish>Prędkość celowania</Polish>
```

File: addons/attributes/stringtable.xml [hunk 11]
```diff
@@ -82,34 +91,42 @@
...
         </Key>
         <Key ID="STR_ZEN_Attributes_MedicalTraining">
             <English>Medical Training</English>
             <German>Medizinische Ausbildung</German>
+            <Polish>Szkolenie Medyczne</Polish>
         </Key>
         <Key ID="STR_ZEN_Attributes_EngineeringSkill">
             <English>Engineering Skill</English>
             <German>Technische Fähigkeiten</German>
+            <Polish>Umiejętność w inżynierii </Polish>
```
Comment (veteran29) [inline addons/attributes/stringtable.xml]:
```suggestion
            <Polish>Szkolenie Inżynieryjne</Polish>
```

File: addons/attributes/stringtable.xml [hunk 12]
```diff
@@ -82,34 +91,42 @@
...
         </Key>
         <Key ID="STR_ZEN_Attributes_AdvEngineer">
             <English>Adv. Engineer</English>
             <German>Erfahrene Techniker</German>
+            <Polish>Zaawansowany Inżynier</Polish>
         </Key>
         <Key ID="STR_ZEN_Attributes_EOD">
             <English>EOD</English>
             <German>Kampfmittelbeseitigung</German>
+            <Polish>Saperska Jednostka Specjalna</Polish>
```
Comment (veteran29) [inline addons/attributes/stringtable.xml]:
```suggestion
            <Polish>Saper</Polish>
```

File: addons/editor/stringtable.xml [hunk 13]
```diff
@@ -74,10 +74,12 @@
         <Key ID="STR_ZEN_Editor_EjectPassengers">
             <English>Eject Passengers</English>
             <German>Passagiere rauswerfen</German>
+            <Polish>Wysadz pasażerów</Polish>
```
Comment (veteran29) [inline addons/editor/stringtable.xml]:
```suggestion
            <Polish>Wysadź pasażerów</Polish>
```

File: addons/editor/stringtable.xml [hunk 14]
```diff
@@ -74,10 +74,12 @@
...
         </Key>
         <Key ID="STR_ZEN_Editor_EjectPassengers_Description">
             <English>Ejects the passengers of all selected vehicles. If the vehicle is currently flying, the units will be given parachutes.\nUnlike vanilla eject, units will disembark one after another (not immediately, with a short delay in-between).</English>
             <German>Wirft die Fahrgäste aller ausgewählten Fahrzeuge raus. Wenn das Fahrzeug gerade fliegt, erhalten die Einheiten Fallschirme.\nIm Gegensatz zu "Vanilla" sitzen die Einheiten nacheinander ab (nicht alle sofort, sondern mit einer kurzen Verzögerung dazwischen).</German>
+            <Polish>Wysadza pasażerów ze wszystkich wybranych pojazdów. Jeżeli pojazd jest w powietrzu, pasażerowie dostaną spadochron. \nJednostki zostaną wysadzone jeden po drugim(Nie wszyskie na raz). </Polish>
```
Comment (veteran29) [inline addons/editor/stringtable.xml]:
```suggestion
            <Polish>Wysadza pasażerów ze wszystkich wybranych pojazdów. Jeżeli pojazd jest w powietrzu, pasażerowie dostaną spadochron.\nJednostki zostaną wysadzone jedna po drugiej (nie wszystkie na raz).</Polish>
```
Trailing space, typos

File: addons/modules/stringtable.xml [hunk 15]
```diff
@@ -256,10 +258,12 @@
         <Key ID="STR_ZEN_Modules_ModuleAmbientFlyby_Amount">
             <English>Amount</English>
             <German>Menge</German>
+            <Polish>Liczba</Polish>
```
Comment (veteran29) [inline addons/modules/stringtable.xml]:
```suggestion
            <Polish>Ilość</Polish>
```
Fits better for the context

File: addons/modules/stringtable.xml [hunk 16]
```diff
@@ -256,10 +258,12 @@
...
         </Key>
         <Key ID="STR_ZEN_Modules_ModuleAmbientFlyby_Amount_Tooltip">
             <English>Sets the amount of aircraft that will be spawned.</English>
             <German>Legt die Menge an Luftfahrzeugen fest, die erzeugt werden sollen.</German>
+            <Polish>Ustala liczbę samolotów  </Polish>
```
Comment (veteran29) [inline addons/modules/stringtable.xml]:
```suggestion
            <Polish>Ilość samolotów które zostaną stworzone.</Polish>
```

File: addons/modules/stringtable.xml [hunk 17]
```diff
@@ -785,6 +789,7 @@
         <Key ID="STR_ZEN_Modules_ModuleTeleportPlayers_IncludeVehicles">
             <English>Include Vehicles</English>
             <German>Fahrzeuge einbeziehen</German>
+            <Polish>Wliczając pojazdy</Polish>
```
Comment (veteran29) [inline addons/modules/stringtable.xml]:
```suggestion
            <Polish>Razem z pojazdami</Polish>
```

File: addons/modules/stringtable.xml [hunk 18]
```diff
@@ -1214,18 +1219,22 @@
...
         </Key>
         <Key ID="STR_ZEN_Modules_ModuleWeather_Overcast">
             <English>Overcast</English>
             <German>Bewölkung</German>
+            <Polish>Pochmurność</Polish>
         </Key>
         <Key ID="STR_ZEN_Modules_ModuleWeather_Overcast_Tooltip">
             <English>Sets the amount of cloud cover, higher values result in a greater likelihood of rain/storms and faster wind speeds.</English>
             <German>Legt die Bedeckung der Wolken fest, höhere Werte führen zu einer höheren Wahrscheinlichkeit von Regen/Stürmen und schnelleren Windgeschwindigkeiten.</German>
+            <Polish>Ustala pochmurność, wyska wartość powoduje większe prawdopodobieństwo deszczu/burzy i szybszego wiatru.</Polish>
```
Comment (veteran29) [inline addons/modules/stringtable.xml]:
```suggestion
            <Polish>Ustala pochmurność, wysoka wartość powoduje większe prawdopodobieństwo deszczu/burzy i szybszego wiatru.</Polish>
```

File: addons/modules/stringtable.xml [hunk 19]
```diff
@@ -1282,10 +1299,12 @@
...
             <German>Nebeldichte</German>
+            <Polish>Gęstość mgły</Polish>
         </Key>
         <Key ID="STR_ZEN_Modules_ModuleWeather_FogDensity_Tooltip">
             <English>Sets the fog density at the base fog altitude.</English>
             <German>Stellt die Nebeldichte auf die Basisnebelhöhe ein.</German>
+            <Polish>Ustala gęstość mgły na podstawowej wysokości mgły.</Polish>
         </Key>
         <Key ID="STR_ZEN_Modules_ModuleWeather_FogDecay">
             <English>Fog Decay</English>
```
Comment (veteran29) [inline addons/modules/stringtable.xml]: `Zanik mgły`

File: addons/modules/stringtable.xml [hunk 20]
```diff
@@ -1367,6 +1389,7 @@
             <Japanese>完全な武器庫を追加</Japanese>
             <Chinesesimp>添加完整军火库</Chinesesimp>
             <Chinese>添加完整的軍火庫</Chinese>
+            <Polish>Dodaj cały arsenał</Polish>
```
Comment (veteran29) [inline addons/modules/stringtable.xml]:
```suggestion
            <Polish>Dodaj pełny arsenał</Polish>
```

File: addons/music/stringtable.xml [hunk 21]
```diff
@@ -3,9 +3,11 @@
     <Package name="Music">
         <Key ID="STR_ZEN_Music_ThisIsWarPartOne">
             <English>This Is War - Part 1</English>
+            <Polish>To jest Wojna - Część 1</Polish>
         </Key>
         <Key ID="STR_ZEN_Music_ThisIsWarPartTwo">
             <English>This Is War - Part 2</English>
+            <Polish>To jest Wojna - Część 2</Polish>
```
Comment (veteran29) [inline addons/music/stringtable.xml]: These are music track titles, not sure if they should be translated.

File: addons/visibility/stringtable.xml [hunk 22]
```diff
@@ -4,14 +4,17 @@
         <Key ID="STR_ZEN_Visibility_Enabled">
             <English>Visibility Indicator</English>
             <German>Sichtbarkeitsindikator</German>
+            <Polish>Wskażnik widoczności</Polish>
         </Key>
         <Key ID="STR_ZEN_Visibility_Enabled_Description">
             <English>Indicates when a player can see the Zeus' cursor.</English>
             <German>Zeigt an, wann ein Spieler den Zeuszeiger sehen könnte.</German>
+            <Polish>Wskazuje kiedy kursor Zeusa jest widoczny.</Polish>
```
Comment (veteran29) [inline addons/visibility/stringtable.xml]:
```suggestion
            <Polish>Wskazuje czy pozycja kursora Zeusa jest widoczna dla gracza.</Polish>
```

File: addons/attributes/stringtable.xml [hunk 23]
```diff
@@ -38,38 +38,47 @@
...
         </Key>
         <Key ID="STR_ZEN_Attributes_AimingSpeed">
             <English>Aiming Speed</English>
             <German>Zilelgeschwindigkeit</German>
+            <Polish>Prędkość Wycelowania</Polish>
         </Key>
         <Key ID="STR_ZEN_Attributes_AimingShake">
             <English>Aiming Shake</English>
             <German>Waffenschwankung</German>
+            <Polish>Dokładność Wycelowania</Polish>
```
Comment (veteran29) [inline addons/attributes/stringtable.xml]:
```suggestion
            <Polish>Niestabilność celowania</Polish>
```

File: addons/modules/stringtable.xml [hunk 24]
```diff
@@ -1376,18 +1399,22 @@
             <Japanese>武器庫を削除</Japanese>
             <Chinesesimp>移除军火库</Chinesesimp>
             <Chinese>移除軍火庫</Chinese>
+            <Polish>Usuń arsenał</Polish>
         </Key>
         <Key ID="STR_ZEN_Modules_ModuleCreateIntel">
             <English>Create/Edit Intel</English>
+            <Polish>Stwórz/Edytuj tajną informację</Polish>
```
Comment (veteran29) [inline addons/modules/stringtable.xml]:
```suggestion
            <Polish>Stwórz/Edytuj dane wywiadowcze</Polish>
```

File: addons/modules/stringtable.xml [hunk 25]
```diff
@@ -1376,18 +1399,22 @@
...
         </Key>
         <Key ID="STR_ZEN_Modules_ModuleCreateIntel_ObjectType">
             <English>Object Type</English>
+            <Polish>Typ Obiektu</Polish>
         </Key>
         <Key ID="STR_ZEN_Modules_ModuleCreateIntel_ShareWith">
             <English>Share With</English>
```
Comment (veteran29) [inline addons/modules/stringtable.xml]: `Współdziel z`

File: addons/modules/stringtable.xml [hunk 26]
```diff
@@ -1376,18 +1399,22 @@
...
         </Key>
         <Key ID="STR_ZEN_Modules_ModuleCreateIntel_DeleteOnCompletion">
             <English>Delete On Completion</English>
+            <Polish>Usuń przy zakończeniu</Polish>
```
Comment (veteran29) [inline addons/modules/stringtable.xml]:
```suggestion
            <Polish>Usuń po zakończeniu</Polish>
```

File: addons/modules/stringtable.xml [hunk 27]
```diff
@@ -1376,18 +1399,22 @@
...
         </Key>
         <Key ID="STR_ZEN_Modules_ModuleCreateIntel_ActionText">
             <English>Action Text</English>
```
Comment (veteran29) [inline addons/modules/stringtable.xml]: `Tekst akcji`

File: addons/modules/stringtable.xml [hunk 28]
```diff
@@ -1397,15 +1424,19 @@
         </Key>
         <Key ID="STR_ZEN_Modules_ModuleCreateIntel_IntelTitle">
             <English>Intel Title</English>
+            <Polish>Tytuł tajnej informacji</Polish>
```
Comment (veteran29) [inline addons/modules/stringtable.xml]:
```suggestion
            <Polish>Tytuł danych wywiadowczych</Polish>
```

File: addons/modules/stringtable.xml [hunk 29]
```diff
@@ -1397,15 +1424,19 @@
...
         </Key>
         <Key ID="STR_ZEN_Modules_ModuleCreateIntel_IntelText">
             <English>Intel Text</English>
+            <Polish>Treść tajnej informacji</Polish>
```
Comment (veteran29) [inline addons/modules/stringtable.xml]:
```suggestion
            <Polish>Treść tajnej informacji</Polish>
```

File: addons/modules/stringtable.xml [hunk 30]
```diff
@@ -1397,15 +1424,19 @@
...
         </Key>
         <Key ID="STR_ZEN_Modules_ModuleCreateIntel_PickUpIntel">
             <English>Pick Up Intel</English>
+            <Polish>Podnieś tajne informację</Polish>
```
Comment (veteran29) [inline addons/modules/stringtable.xml]:
```suggestion
            <Polish>Podnieś dane wywiadowcze</Polish>
```

File: addons/modules/stringtable.xml [hunk 31]
```diff
@@ -1397,15 +1424,19 @@
...
         </Key>
         <Key ID="STR_ZEN_Modules_ModuleCreateIntel_IntelFound">
             <English>Intel Found</English>
+            <Polish>Znaleziono tajne informacje</Polish>
```
Comment (veteran29) [inline addons/modules/stringtable.xml]:
```suggestion
            <Polish>Znaleziono dane wywiadowcze</Polish>
```

File: addons/modules/stringtable.xml [hunk 32]
```diff
@@ -1397,15 +1424,19 @@
...
         </Key>
         <Key ID="STR_ZEN_Modules_ModuleCreateIntel_PlayerFoundIntel">
             <English>%1 found intel: "%2"</English>
```
Comment (veteran29) [inline addons/modules/stringtable.xml]: `%1 znalazł dane wywiadowcze: "%2"`

**Comment by veteran29** [comment]
@Maciox55 
You could add suggestions in batch on `Files` tab instead od creating a commit per single one...

File: addons/music/stringtable.xml (hunk 21 above)
Comment (Maciox55) [reply addons/music/stringtable.xml]: True

File: addons/camera/stringtable.xml (hunk 1 above)
Comment (veteran29) [reply addons/camera/stringtable.xml]: This one was marked as resolved but is not fixed.

File: addons/music/stringtable.xml (hunk 21 above)
Comment (veteran29) [reply addons/music/stringtable.xml]: Marked as resolved but music title translations are still there.

**Comment by CreepPork** [comment]
@mharis001 Veteran’s concerns were not addressed fully. PR should be reverted.

**Comment by veteran29** [comment]
No, do not revert, it's minor issues. I can PR improvements soon:tm:

**Comment by mharis001** [comment]
I believe I added all of the suggestions that weren't yet committed.

**Pull request merged**